src/middlewared/middlewared/assets/account/builtin/freebsd/passwd

```diff
@@ -16,6 +16,7 @@ proxy:*:62:62:Packet Filter pseudo-user:/nonexistent:/usr/sbin/nologin
 _pflogd:*:64:64:pflogd privsep user:/var/empty:/usr/sbin/nologin
 _dhcp:*:65:65:dhcp programs:/var/empty:/usr/sbin/nologin
 uucp:*:66:66:UUCP pseudo-user:/var/spool/uucppublic:/usr/local/libexec/uucp/uucico
+nut:*:316:316:nut:/nonexistent:/bin/sh
```

Comment:
Perhaps `/var/empty` instead of `/nonexistent`.

Reply:
Why don't we change the runas user back to UUCP in 13.1? What's motivation for changing user?